### ISSUE TITLE
Add Markdown Export to CPL

### ISSUE THREAD

**Issue by cdaven** [tiddly-gittly/TiddlyWiki-CPL#53]
Please add the Markdown Export plugin to CPL.

JSON file: https://raw.githubusercontent.com/cdaven/tiddlywiki-stuff/main/markdown-export/%24__plugins_cdaven_markdown-export.json

GitHub repo: https://github.com/cdaven/tiddlywiki-stuff/tree/main/markdown-export

**Comment by linonetwo** [comment]
Thanks, added for you. Please try keep the links constant!

**Comment by cdaven** [comment]
I published a new version of the plugin, and the "readme" in CPL doesn't update from the JSON file.

Compare the last parapraphs from the original: https://cdaven.github.io/tiddlywiki/#%24%3A%2Fplugins%2Fcdaven%2Fmarkdown-export

With: https://tw-cpl.netlify.app/#cdaven%2Fmarkdown-export:Index%20cdaven%2Fmarkdown-export

**Comment by linonetwo** [comment]
That is true, it has to be manually rewrite in the CPL site, if you would like, you can fork CPL, update and make a PR.

I'm planning to add a feature that allow author fill in which tiddler is the readme tiddler, then auto extract it from the plugin instead of written by hand every time.
